.drone.yml

```diff
@@ -48,16 +48,18 @@ pipeline:
     commands:
       - du -ks harbor-offline-installer-*.tgz | awk '{print $1 / 1024}' | { read x; echo $x MB; }
       - mkdir -p bundle
-      - mkdir -p latest
-      - echo $(git describe --tags) > latest/version 
+      - mkdir -p pks-bundle
+      - echo $(git describe --tags) > pks-bundle/version 
       - cp harbor-offline-installer-*.tgz bundle
-      - if [ ${DRONE_BRANCH} = "master" ]; then cp harbor-offline-installer-*.tgz latest/harbor-offline-installer-latest-master.tgz; else cp harbor-offline-installer-*.tgz latest/harbor-offline-installer-latest-release.tgz; fi
+      - if [ ${DRONE_BRANCH} = "master" ]; then cp harbor-offline-installer-*.tgz pks-bundle/harbor-offline-installer-latest-master.tgz; fi
+      - if ( echo ${DRONE_BRANCH} | grep "release*" ); then cp harbor-offline-installer-*.tgz pks-bundle/harbor-offline-installer-latest-release.tgz; fi
```

Comment:
Why update pks bundle when there's change in release branch?  If there's a change in pks branch then a change in release 1.3.0 branch, the tile CI should not pick up the installer in release 1.3.0 branch.